### PR TITLE
[Bug] Making text or graphical representation available for the mic on participants

### PR DESCRIPTION
**Pull request by Mohtasim**
## Purpose

* Making text or graphical representation available for the participants, currently communicating, with .MIC ON'.
* bug description: https://skype.visualstudio.com/SPOOL/_workitems/edit/2763088

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Turn on the Talkback
* Navigate to call setup page
* Navigate to Join call and activate it
* Navigate to the floating header and activate participant list
* Navigate in the list with current call setting for each participant
* Observe the issue

## What to Check
Verify that the following are valid
* Participants'(currently communicating) with MIC ON visual or programmatical representation should be present.

### CODE REVIEW

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt [hunk 1]
```diff
@@ -161,16 +161,27 @@ internal class ParticipantListView(
...
+        )
+        else ContextCompat.getDrawable(
+            context,
+            R.drawable.azure_communication_ui_ic_fluent_mic_on_24_filled_composite_button_filled_grey
+        )
+
+        val micAccessibilityAnnouncement = if (isMuted) context.getString(
+            R.string.azure_communication_ui_calling_view_participant_list_muted_accessibility_label
+        )
+        else "Unmuted"
```
Comment (lpx1989) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt:176]: please use referenced string from string.xml

File: docs/api/CallComposite/classes/LocalizationConfiguration.md [hunk 2]
```diff
@@ -16,26 +16,48 @@ Creates an instance of `LocalizationConfiguration` with related parameters.
 
 ```java
 public LocalizationConfiguration(
-    final LanguageCode languageCode
+    final Locale locale
```
Comment (lpx1989) [inline docs/api/CallComposite/classes/LocalizationConfiguration.md:19]: why this PR has changes under LocalizationConfiguration file?

File: docs/api/CallComposite/classes/SupportLanguage.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+**CLASS**
```
Comment (lpx1989) [inline docs/api/CallComposite/classes/SupportLanguage.md:1]: This file is part of this PR change?

Comment (Mohtasim) [reply docs/api/CallComposite/classes/SupportLanguage.md:1]: No. This change came with merge from develop/beta.

File: docs/api/CallComposite/classes/LocalizationConfiguration.md (hunk 2 above)
Comment (Mohtasim) [reply docs/api/CallComposite/classes/LocalizationConfiguration.md:19]: This change came with merge from develop/beta

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt (hunk 1 above)
Comment (Mohtasim) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt:176]: Thanks. I was waiting for the pipeline translations to be made.

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt [hunk 4]
```diff
@@ -161,16 +161,27 @@ internal class ParticipantListView(
         isMuted: Boolean,
         personaData: CommunicationUIPersonaData?,
     ): BottomCellItem {
+        val micIcon = if (isMuted) ContextCompat.getDrawable(
```
Comment (ahammer) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt:164]: Use the if on the R.Drawable resource only (less branching code)

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt [hunk 5]
```diff
@@ -161,16 +161,27 @@ internal class ParticipantListView(
...
+            context,
+            R.drawable.azure_communication_ui_ic_fluent_mic_off_24_filled_composite_button_filled_grey
+        )
+        else ContextCompat.getDrawable(
+            context,
+            R.drawable.azure_communication_ui_ic_fluent_mic_on_24_filled_composite_button_filled_grey
+        )
+
+        val micAccessibilityAnnouncement = if (isMuted) context.getString(
+            R.string.azure_communication_ui_calling_view_participant_list_muted_accessibility_label
```
Comment (ahammer) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/participantlist/ParticipantListView.kt:174]: same, put the if inside context.getString()

File: azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt [hunk 6]
```diff
@@ -73,10 +73,22 @@ internal class BottomCellViewHolder(itemView: View) : RecyclerView.ViewHolder(it
...
         onClickAction = bottomCellItem.onClickAction
     }
+
+    private fun isAccessoryImageViewable(bottomCellItem: BottomCellItem): Boolean {
+        val muteDescription = itemView.rootView.context
+            .getString(R.string.azure_communication_ui_calling_view_participant_list_muted_accessibility_label)
+        val unmutedDescription = itemView.rootView.context
+            .getString(R.string.azure_communication_ui_calling_view_participant_list_unmuted_accessibility_label)
+
+        if (bottomCellItem.enabled == true) return true
```
Comment (ahammer) [inline azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt:88]: Use {} for this branching if/else/elseif

Comment (ahammer) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt:88]: also, just need one if and ||, not a else if.

Comment (ahammer) [reply azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellViewHolder.kt:88]: can actually just return the boolean.

e.g. return   (bottomCellItem.enabled || bottomCellItem.accessoryImageDescription == muteDescription || bottomCellItem.accessoryImageDescription == unmutedDescription)

**Review by lpeixin — APPROVED** [review]
Tested

**Review by ahammer — APPROVED** [review]

**Review by Mrayyan — APPROVED** [review]

**Pull request merged**